UnitsNet/CustomCode/UnitAbbreviationsCache.cs

```diff
@@ -300,7 +300,7 @@ internal bool TryGetUnitValueAbbreviationLookup(Type unitType, IFormatProvider?
         {
             unitToAbbreviations = null;
 
-            formatProvider ??= CultureInfo.CurrentUICulture;
+            formatProvider = formatProvider ?? CultureInfo.CurrentCulture;
```

Comment:
Is there a reason you reverted this syntax? Not significant, just curious.

Reply:
Probably a merge conflict of some kind.